app/models/inquiry/column.rb

```diff
@@ -83,4 +85,22 @@ def validate_check_box(answer, data)
       end
     end
   end
+
+  def validate_upload_file(answer, data)
+    # MegaBytes >> Bytes
+    if self.max_upload_file_size.to_i > 0 
+      fileSize  = data.values[2].to_i
+      limitSize = (self.max_upload_file_size * 1024 * 1024).to_i

```

Comment:
__[rubocop]__
* Use snake_case for variable names. [:ref](https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)

<!-- Comment by [SideCI](https://sideci.com) -->